docs/pages/guides/environment-variables.mdx

```diff
@@ -59,16 +61,31 @@ If you create an environment-specific file, like **.env.test**, you can load it
 
 <Terminal cmd={['$ NODE_ENV=production npx expo start']} />
 
+### Disabling environment variables
+
+Environment variables in Expo CLI have two parts, both can be disabled:
+
+1. Expo CLI will load the `.env` files into the global process. This can be disabled by setting the environment variable `EXPO_NO_DOTENV` before any Expo CLI command is run, like `EXPO_NO_DOTENV=1`.
+2. Expo's Metro config will inline environment variables in the client JavaScript bundle during serialization. This can be disabled with `EXPO_NO_CLIENT_ENV_VARS`, like `EXPO_NO_CLIENT_ENV_VARS=1`.
```

Comment:
```suggestion
2. Expo's Metro config includes the inline serialization of environment variables in the client JavaScript bundle. To disable this behavior, you can use `EXPO_NO_CLIENT_ENV_VARS=1`.
```

- `inline environment variables in the client` was confusing as to what does inline meant in that context. Tried to update the phrase as above.
- one thing missing from above is whether `EXPO_NO_CLIENT_ENV_VARS=1` has to be run before an Expo CLI command (like in point 1)?